src/SesWebhookCall.php

```diff
@@ -20,4 +20,15 @@ public static function storeWebhook(WebhookConfig $config, Request $request): We
             'payload' => $message->toArray(),
         ]);
     }
+
+    public static function getFirstIdOfThisSesMessage(string $ses_message_id): int
+    {
+        return SesWebhookCall::where('payload->MessageId', $ses_message_id)
+                                ->min('id');
+    }
+
+    public function isFirstOfThisSesMessage(): bool
```

Comment:
This can be private

Reply:
getFirstIdOfThisSesMessage() no longer exists.